Mathlib/Analysis/Calculus/FDeriv/Analytic.lean

```diff
@@ -393,6 +395,7 @@ theorem AnalyticOnNhd.iteratedFDeriv_of_isOpen (h : AnalyticOnNhd 𝕜 f s) (hs
 
 /-- If a partial homeomorphism `f` is analytic at a point `a`, with invertible derivative, then
 its inverse is analytic at `f a`. -/
+@[fun_prop]
```

Comment:
I think this one and the next one shouldn't be tagged either, because there's no way fun_prop will  find that the derivative is invertible.

Reply:
Roger. Remove the fun_prop as requested.